### ISSUE TITLE
How to properly train the exact same pretrained model from the published dataset?

### ISSUE THREAD

**Issue by AnkBurov** [DoubangoTelecom/tesseractMRZ#3]
Hello. I checked https://github.com/DoubangoTelecom/tesseractMRZ/issues/2 issue and used https://github.com/tesseract-ocr/tesstrain project to train the model from your dataset. I also used the exact same neural network architecture, but the resulting accuracy of the trained model is rather poor and totally incomparable with this pretrained_model: https://github.com/DoubangoTelecom/tesseractMRZ/blob/master/tessdata_best/mrz.traineddata (40% accuracy against 94% in my test environment).
The training script I use:
```
make training MODEL_NAME=mrz_new NET_SPEC="[1,36,0,1 Ct3,3,16 Mp3,3 Lfys64 Lfx96 Lrx96 Lfx512 O1c39]"
```
and the dataset I use is the dataset from this project.
I understand that some variations in accuracy are inevitable due to the random nature of training data shuffling for train / eval division, but the error cannot be that big.

I feel like I'm missing something crucial, so is the training script invocation looks valid enough?

**Comment by AnkBurov** [comment]
Okay, I think I know where is the problem. I didn't generate `.box` files with correct coordinates of each symbol while `tesstrain` generates `.box` files with 0-0 coordinates because it obviously doesn't know their coordinates. Closing, thanks for the model! :smile:
